### ISSUE TITLE
Alerts tab page

### ISSUE THREAD

**Issue by robertsteilberg** [mVax-app/mVax#18]
Alerts tab does not accurately reflect the goal of the tab and is somewhat confusing to users. Currently used for overdue patients, doesn't really make any alerts per se.

Acceptance criteria: Redefine what the tab should be named and make sure that the UI reflects this change.

**Comment by stevenyang9** [comment]
Create a calendar that indicates patients that are due for a vaccination on each day.
